itest/psbt_test.go

```diff
@@ -1108,6 +1150,82 @@ func testMultiInputPsbtSingleAssetID(t *harnessTest) {
 	var foundAssetId asset.ID
 	copy(foundAssetId[:], primaryNodeAsset.AssetGenesis.AssetId)
 	require.Equal(t.t, assetId, foundAssetId)
+
+	t.Logf("Partial amount multi-input PSBT send event complete. Now " +
+		"attempting to send the remaining amount in a full value " +
+		"multi-input PSBT send event back to the primary node from " +
+		"the secondary node.")
+
+	// Attempt a full value send of the rest of the asset back to the
+	// primary node.
+	sendAmt = uint64(1500)
+
+	vPkt = tappsbt.ForInteractiveSend(
```

Comment:
super-nit: we could assert the # of inputs on these PSBTs;  IIUC should be two in both cases?

Reply:
good idea, added